packages/c/carbon-charts.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "carbon-charts",
+  "description": "Carbon Charts component library",
+  "keywords": [
+		"charts",
+		"graphs",
+		"radar",
+		"gauge",
+		"donut",
+		"pie",
+		"sparkline",
+		"tree",
+		"treemap",
+		"heatmap",
+		"wordcloud",
+		"histogram",
+		"alluvial",
+		"geo",
+		"bar",
+		"bullet",
+		"scatter",
+		"meter",
+		"line",
+		"javascript",
+		"component",
+		"carbon",
+		"ibm",
+		"svg",
+		"data",
+		"typescript"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://www.carbondesignsystem.com/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core"
+  },
+  "filename": "dist/umd/bundle.umd.js",
```

```suggestion
  "filename": "bundle.umd.js",
```
As you're setting `basePath`, we'll pull files up out of that directory